### ISSUE TITLE
Conflicting guidance on aria-roledescription for Tabbed Carousel pattern

### ISSUE THREAD

**Issue by grelas** [w3c/aria-practices#1527]
In this [Tabbed Carousel example](https://w3c.github.io/aria-practices/examples/carousel/carousel-2-tablist.html?moreaccessible=true), the `tabpanel` section indicates that `aria-roledescription="slide"` be present on the element.

However, this conflicts with the description under the `Tabbed Carousel Elements` on the [Authoring Practices guide](https://w3c.github.io/aria-practices/#tabbed-carousel-elements), which states:

> Each slide container has role tabpanel in lieu of group, and it does not have the aria-roledescription property.

So the main question here is, should we be adding `aria-roledescription="slide"` on the `tabpanel` slide container?
